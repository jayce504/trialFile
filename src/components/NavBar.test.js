import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';



describe('NavBar', () => {
    const navbar = shallow(<NavBar />);
    const request = require('supertest');
    const app = require('./app');

    it('renders properly', () => {
        expect(navbar).toMatchSnapshot();
    })

});

