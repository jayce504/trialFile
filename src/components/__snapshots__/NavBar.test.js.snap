// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav
        className="navbar navbar-expand-lg"
      >
        <a
          className="navbar-brand"
          href="/"
        >
          TrialFile
        </a>
        <button
          aria-controls="navbarNav"
          aria-expanded="false"
          aria-label="Toggle navigation"
          className="navbar-toggler"
          data-target="#navbarNav"
          data-toggle="collapse"
          type="button"
        >
          <span
            className="navbar-toggler-icon"
          />
        </button>
        <div
          className="collapse navbar-collapse"
          id="navbarNav"
        >
          <ul
            className="navbar-nav"
          >
            <li
              className="nav-item active"
            >
              <a
                className="nav-link"
                href="/logout"
              >
                Logout 
                <span
                  className="sr-only"
                >
                  (current)
                </span>
              </a>
            </li>
          </ul>
        </div>
      </nav>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            className="navbar-brand"
            href="/"
          >
            TrialFile
          </a>,
          <button
            aria-controls="navbarNav"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-target="#navbarNav"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>,
          <div
            className="collapse navbar-collapse"
            id="navbarNav"
          >
            <ul
              className="navbar-nav"
            >
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/logout"
                >
                  Logout 
                  <span
                    className="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
            </ul>
          </div>,
        ],
        "className": "navbar navbar-expand-lg",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "TrialFile",
            "className": "navbar-brand",
            "href": "/",
          },
          "ref": null,
          "rendered": "TrialFile",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-controls": "navbarNav",
            "aria-expanded": "false",
            "aria-label": "Toggle navigation",
            "children": <span
              className="navbar-toggler-icon"
            />,
            "className": "navbar-toggler",
            "data-target": "#navbarNav",
            "data-toggle": "collapse",
            "type": "button",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "navbar-toggler-icon",
            },
            "ref": null,
            "rendered": null,
            "type": "span",
          },
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul
              className="navbar-nav"
            >
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/logout"
                >
                  Logout 
                  <span
                    className="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
            </ul>,
            "className": "collapse navbar-collapse",
            "id": "navbarNav",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/logout"
                >
                  Logout 
                  <span
                    className="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>,
              "className": "navbar-nav",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  className="nav-link"
                  href="/logout"
                >
                  Logout 
                  <span
                    className="sr-only"
                  >
                    (current)
                  </span>
                </a>,
                "className": "nav-item active",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Logout ",
                    <span
                      className="sr-only"
                    >
                      (current)
                    </span>,
                  ],
                  "className": "nav-link",
                  "href": "/logout",
                },
                "ref": null,
                "rendered": Array [
                  "Logout ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "(current)",
                      "className": "sr-only",
                    },
                    "ref": null,
                    "rendered": "(current)",
                    "type": "span",
                  },
                ],
                "type": "a",
              },
              "type": "li",
            },
            "type": "ul",
          },
          "type": "div",
        },
      ],
      "type": "nav",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav
          className="navbar navbar-expand-lg"
        >
          <a
            className="navbar-brand"
            href="/"
          >
            TrialFile
          </a>
          <button
            aria-controls="navbarNav"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-target="#navbarNav"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse"
            id="navbarNav"
          >
            <ul
              className="navbar-nav"
            >
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/logout"
                >
                  Logout 
                  <span
                    className="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
            </ul>
          </div>
        </nav>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              className="navbar-brand"
              href="/"
            >
              TrialFile
            </a>,
            <button
              aria-controls="navbarNav"
              aria-expanded="false"
              aria-label="Toggle navigation"
              className="navbar-toggler"
              data-target="#navbarNav"
              data-toggle="collapse"
              type="button"
            >
              <span
                className="navbar-toggler-icon"
              />
            </button>,
            <div
              className="collapse navbar-collapse"
              id="navbarNav"
            >
              <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/logout"
                  >
                    Logout 
                    <span
                      className="sr-only"
                    >
                      (current)
                    </span>
                  </a>
                </li>
              </ul>
            </div>,
          ],
          "className": "navbar navbar-expand-lg",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "TrialFile",
              "className": "navbar-brand",
              "href": "/",
            },
            "ref": null,
            "rendered": "TrialFile",
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "aria-controls": "navbarNav",
              "aria-expanded": "false",
              "aria-label": "Toggle navigation",
              "children": <span
                className="navbar-toggler-icon"
              />,
              "className": "navbar-toggler",
              "data-target": "#navbarNav",
              "data-toggle": "collapse",
              "type": "button",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "navbar-toggler-icon",
              },
              "ref": null,
              "rendered": null,
              "type": "span",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/logout"
                  >
                    Logout 
                    <span
                      className="sr-only"
                    >
                      (current)
                    </span>
                  </a>
                </li>
              </ul>,
              "className": "collapse navbar-collapse",
              "id": "navbarNav",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/logout"
                  >
                    Logout 
                    <span
                      className="sr-only"
                    >
                      (current)
                    </span>
                  </a>
                </li>,
                "className": "navbar-nav",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    className="nav-link"
                    href="/logout"
                  >
                    Logout 
                    <span
                      className="sr-only"
                    >
                      (current)
                    </span>
                  </a>,
                  "className": "nav-item active",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      "Logout ",
                      <span
                        className="sr-only"
                      >
                        (current)
                      </span>,
                    ],
                    "className": "nav-link",
                    "href": "/logout",
                  },
                  "ref": null,
                  "rendered": Array [
                    "Logout ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "(current)",
                        "className": "sr-only",
                      },
                      "ref": null,
                      "rendered": "(current)",
                      "type": "span",
                    },
                  ],
                  "type": "a",
                },
                "type": "li",
              },
              "type": "ul",
            },
            "type": "div",
          },
        ],
        "type": "nav",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
